src/js/view/controls/controls.js

```diff
@@ -110,7 +110,7 @@ export default class Controls extends Events {
                 this.trigger(DISPLAY_CLICK);
                 this.userActive(1000);
                 api.playToggle(reasonInteraction());
-                this.playerContainer.focus();
+                model.get('isFloating') ? this.wrapperElement.focus() : this.playerContainer.focus();
```

Comment:
Since the wrapper is always in the player, couldn't we just always focus the wrapper?